homeassistant/components/conversation/__init__.py

```diff
@@ -36,15 +36,7 @@
 )
 
 CONFIG_SCHEMA = vol.Schema(
-    {
-        DOMAIN: vol.Schema(
-            {
-                vol.Optional("intents"): vol.Schema(
-                    {cv.string: vol.All(cv.ensure_list, [cv.string])}
-                )
-            }
-        )
-    },
+    {DOMAIN: vol.Schema({})},
```

Comment:
We should print a warning that it's no longer supported. In a few releases we can completely remove `CONFIG_SCHEMA`

Reply:
Printing a warning if there's anything in the config.